esp-wifi/src/lib.rs

```diff
@@ -153,6 +153,10 @@ struct Config {
     heap_size: usize,
     #[default(DEFAULT_TICK_RATE_HZ)]
     tick_rate_hz: u32,
+    #[default(3)]
+    listen_interval: u16,
+    #[default(6)]
```

Comment:
@bjoernQ If this default is used, starting AP panics. Quoting from https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/network/esp_wifi.html#_CPPv426esp_wifi_set_inactive_time16wifi_interface_t8uint16_t

> ESP_ERR_INVALID_ARG: invalid argument, For Station, if sec is less than 3. For SoftAP, if sec is less than 10.

I believe a different config should be provided for AP and STA modes.